datawell/iex.py

```diff
@@ -56,3 +57,28 @@ def load_from_iex(self, uri: str):
             error = response.status_code
             self.Logger.error(
                 f'Encountered an error: {error} ( {response.text} ) while retrieving {uri}')
+    
+    def getCompany(self, *Symbols):
+        """
+        Will return companies according to the stocks or all stocks.
+        https://github.com/petelind/ConsumingAPI/issues/2
+        TODO:  ADD Cache for companies
+        TODO:  Add special symbol handler(*,-,+ etc.)
+        TODO:  ADD multithreads (takes 1h40m without)
```

Comment:
All ideas very good, but I'd suggest you leave TODOs out of the master codes - we typically employ those freely, but it seems like the general convention they are for the "work in progress". (If you'll have a look on the linter output and commit checks in PyCharm, you'll notice that they are frowning on the TODOs in the commit).